src/app/shared/components/strength-meter/password-strength.ts

```diff
@@ -0,0 +1,70 @@
+/* @format */
+import { Component, Input, OnChanges } from '@angular/core';
+import { FeatureFlagService } from '@root/app/feature-flag/services/feature-flag.service';
+import { passwordStrength } from 'check-password-strength';
+
+enum PasswordStrength {
+  Null = 'null',
+  Weak = 'weak',
+  Medium = 'medium',
+  Strong = 'strong',
+}
+
+@Component({
+  selector: 'pr-password-strength',
+  templateUrl: './password-strength.html',
+  styleUrls: ['./password-strength.scss'],
+})
+export class PasswordStrengthComponent implements OnChanges {
+  progressBars: string[] = ['', '', ''];
+
+  public strength: PasswordStrength = PasswordStrength.Null;
+  public message: string = '';
+  public passwordClass: string = '';
+  @Input() password: string = '';
+  public enabledPasswordCheckStrength: boolean;
```

Comment:
There's a bit of style inconsistency here. Generally we like to visually separate the Input() fields from standard class properties. And some of these have access modifiers (something I generally like to see) while some don't.
```suggestion
  @Input() public password: string = '';
  
  public progressBars: [string, string, string] = ['', '', ''];
  public strength: PasswordStrength = PasswordStrength.Null;
  public message: string = '';
  public passwordClass: string = '';
  public enabledPasswordCheckStrength: boolean;
```